src/main/java/com/microsoft/aad/msal4j/IClientApplicationBase.java

```diff
@@ -92,6 +92,15 @@
      */
     CompletableFuture<IAuthenticationResult> acquireToken(AuthorizationCodeParameters parameters);
 
+    /**
+     * Acquires a security token from the authority using a Refresh Token previously received.
+     *
+     * @param parameters#refreshToken Refresh Token to use in the refresh flow.
+     * @param parameters#scopes       scopes of the access request
+     * @return A {@link CompletableFuture} object representing the {@link IAuthenticationResult} of the call.
+     */
+    CompletableFuture<IAuthenticationResult> acquireToken(RefreshTokenParameters parameters);
```

Comment:
acquireTokenByRefreshToken + see comments in other libraries which also have aka.ms links to additional details.

Reply:
thanks, for pointing out , have copy pasted